proof_ipa.go

```diff
@@ -83,17 +82,20 @@ type Proof struct {
 	PostValues [][]byte
 }
 
-type SuffixStateDiff struct {
```

Comment:
Something we should remember, this kind of new format requires more integrity checks to see if same suffixes aren't claimed to be new, updated and/or untouched at the same time.